sdks/uniswapx-sdk/integration/test/V3DutchOrder.spec.ts

```diff
@@ -303,17 +311,436 @@ describe.only("DutchV3Order", () => {
         );
     });
 
-    const getCosignerData = async (
-        overrides: Partial<V3CosignerData> = {}
-        ): Promise<V3CosignerData> => {
-        const defaultData: V3CosignerData = {
-            decayStartBlock: 20916059,
-            exclusiveFiller: ethers.constants.AddressZero,
-            exclusivityOverrideBps: BigNumber.from(0),
-            // overrides of 0 will not affect the values
-            inputOverride: BigNumber.from(0),
-            outputOverrides: [BigNumber.from(0)],
-        };
-        return Object.assign(defaultData, overrides);
-        };
-});
\ No newline at end of file
+    it("executes a serialized order with no decay, override of double original output amount", async () => {
+        const deadline = await new BlockchainTime().secondsFromNow(1000);
+        const order = new V3DutchOrderBuilder(
+            chainId,
+            reactor.address,
+            permit2.address
+        )
+            .cosigner(cosigner.address)
+            .deadline(deadline)
+            .swapper(swapper.address)
+            .nonce(NONCE)
+            .startingBaseFee(BigNumber.from(0))
+            .input({
+                token: tokenIn.address,
+                startAmount: SMALL_AMOUNT,
+                curve: {
+                    relativeBlocks: [1],
+                    relativeAmounts: [BigInt(0)],
+                },
+                maxAmount: SMALL_AMOUNT,
+                adjustmentPerGweiBaseFee: BigNumber.from(0),
+            })
+            .output({
+                token: tokenOut.address,
+                startAmount: SMALL_AMOUNT,
+                curve: {
+                    relativeBlocks: [1],
```

Comment:
No decay can also be achieved with two empty arrays and I think in practice that's preferable for gas savings.